desktop/release-notes.md

```diff
@@ -24,12 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- If Docker Desktop detects that the HTTP or HTTPS proxy password has changed then it will prompt developers for the new password
+- The "bypass proxy settings for these hosts and domains" setting now handles domain names correctly for HTTPS
+- Added a user setting to opt out of 4.13-specific features that are in development.
+- Added categories to the Extensions Marketplace.
+- Added an indicator in the whale menu and on the **Extension** tab on when extension updates are available.
+- Fixed failing uninstalls of extensions with image names that do not have a namespace, as in 'my-extension'.
+- Show port mapping explicitly in the **Container** tab.
+- Changed the refresh rate for disk usage information for images to happen automatically once a day.
+- Made the tab style consistent for the **Container** and **Volume** tabs.
+- Fixed Grpcfuse filesharing mode enablement in **Settings**.
+- Virtualization Framework and VirtioFS are disabled for users running macOS < 12.5.
+- Ports on the **Containers** tab are now clickable.
+- The Extensions SDK now allows `ddClient.extension.vm.cli.exec`, `ddClient.extension.host.cli.exec`, `ddClient.docker.cli.exec` to accept a different working directory and pass environment variables through the options parameters.
+- Added a small improvement to navigate to the Extensions Marketplace when clicking on **Extensions** in the sidebar.
+- Added a badge to identify new extensions in the Marketplace.
+- Fixed kubernetes not starting with the containerd integration.
+- Fixed `kind` not starting with the containerd integration.
+- Fixed dev environments not working with the containerd integration.
+- Implemented `docker diff` in the containerd integration.
+- Implemented `docker run —-platform` in the containerd integration.
+- Fixed insecure registries not working with the containerd integration.
+- Fixed a bug that showed incorrect values on used space in **Settings**.
+
+#### For Mac
+
+- Added **Hide**, **Hide others**, **Show all** menu items for Docker Desktop. See [docker/for-mac#6446](https://github.com/docker/for-mac/issues/6446).
+- Fixed a bug which caused the application to be deleted when running the install utility from the installed application. Fixes [docker/for-mac#6442](https://github.com/docker/for-mac/issues/6442).
+
 ## Docker Desktop 4.12.0
 2022-09-01
 
 > Download Docker Desktop
 >
-> {% include desktop-install.html %}
+> [Windows](https://desktop.docker.com/win/main/amd64/85629/Docker%20Desktop%20Installer.exe) |
+> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/85629/Docker.dmg) |
+> [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/85629/Docker.dmg) |
+> [Debian](https://desktop.docker.com/linux/main/amd64/85629/docker-desktop-4.12.0-amd64.deb) |
+> [RPM](https://desktop.docker.com/linux/main/amd64/85629/docker-desktop-4.12.0-x86_64.rpm) |
+> [Arch package](https://desktop.docker.com/linux/main/amd64/85629/docker-desktop-4.12.0-x86_64.pkg.tar.zst)
 
```

Comment:
The checksums for the 4.12 need to added in the same we have for 4.11.1.

Reply:
Done!